cli/enry/main.go

```diff
@@ -77,8 +80,17 @@ func main() {
 		log.Fatal(err)
 	}
 
-	data, _ := json.MarshalIndent(out, "", "  ")
-	fmt.Printf("%s\n", data)
+	if *jsonFlag && !*breakdownFlag {
```

Comment:
I am doing these extrange ifs because linguist flags work weird, if you have JSON and breakdown flags active, only breakdown output is printed. If you have only breakdown flag active, percent output and breakout output are printed.